### DISCUSSION TITLE
CloudSpanner

### DISCUSSION THREAD

**Discussion by jamie-r-sleuth** [janko/sequel-activerecord_connection#26] [Ideas]
I love this Gem, I've used it in two projects so far and working on the third.
Do you know if there are plans to support connections to Google's CloudSpanner ActiveRecord Adatper?
https://cloud.google.com/blog/topics/developers-practitioners/scale-your-ruby-applications-active-record-support-cloud-spanner

**Comment by janko** [comment]
I'm glad to hear you've found the gem useful 🤘

It would be possible to add support only if Sequel had a corresponding adapter as well, but that doesn't seem to be the case.
